### ISSUE TITLE
Can't Share Animated QR of Seed

### ISSUE THREAD

**Issue by shannona** [BlockchainCommons/GordianSeedTool-iOS#137]
If I want to save the QR of a seed, from Authenticate->Backup->ur:crypto-seed, I just long-hold the QR and eventually get a share seed.

However, if that QR is animated, I long-hold and nothing happens.

**Comment by shannona** [comment]
This is also true for the animated QR of the account descriptor; I'd suspect it's true for all animated QRs.

**Comment by ChristopherA** [comment]
Interesting problem here. Longer-term I suppose it could be saved as an animated .png. 

The bigger issue is if can fit into a dense QR, but on the screen it is animated, you should be able to save the dense QR.

**Comment by wolfmcnally** [comment]
This is intentional, as a simple PNG cannot capture an animated QR code, and since the stream of fountain code-based QR codes is essentially infinite, you would rather have to export an animated format such as a video or animated GIF that contained either the first sequence of codes that constitute the entire body of data, or some sequence of those + more fountain codes. But how many? A minimal sequence of codes if played back may not be read correctly. If you loop the GIF you re-introduce the same issue that fountain codes were created to deal with: missing any QR code in the sequence means you need to wait until the sequence loops back to the one you missed, resulting in slower and slower net transmission rate.

To implement this as a feature we would need to decide (or let the user determine):

* Fragment size (maximum density for QR codes)
* Frame rate (how fast we show the QR codes)
* Output format (GIF? Video?)
* Do we include the minimal set of QR codes or a series of fountain codes after them? How many?

Basically, single QR codes have the advantage that they are self contained and can be displayed in any static medium, whether a web page or a printed page. But animated QR codes are for live, real-time transmission of data.

**Comment by wolfmcnally** [comment]
I'm going to close this for now, but if we decide to prioritize it we can open it again.

**Comment by ChristopherA** [comment]
Leaving aside saving it aside as animated, it currently doesn’t allow you to save a pngif it COULD fit in a single QR while printed. Reopening.

**Comment by wolfmcnally** [comment]
That is a separate feature request with its own issues, for example how to properly inform the user that a regular QR code cannot contain it, and how to manage users' expectations that they would actually be receiving an animated QR code. Please open a separate issue for it.

**Comment by ChristopherA** [comment]
I need your help with these. Don’t close until new issue exists.

**Comment by shannona** [comment]
#146 readdresses this per Wolf's notes.
